src/OceanSeaIceModels/InterfaceComputations/interpolate_atmospheric_state.jl

```diff
@@ -3,11 +3,22 @@ using Oceananigans.Grids: _node
 using Oceananigans.Fields: FractionalIndices
 using Oceananigans.OutputReaders: TimeInterpolator
 
-using ...OceanSimulations: forcing_barotropic_potential
+using ...OceanSimulations: forcing_barotropic_potential, TwoColorRadiation
 
 using ClimaOcean.OceanSeaIceModels.PrescribedAtmospheres: PrescribedAtmosphere
 import ClimaOcean.OceanSeaIceModels: interpolate_atmosphere_state!
 
+compute_radiative_forcing!(T_forcing, downwelling_shortwave_radiation, coupled_model) = nothing #fallback
+
+function compute_radiative_forcing!(tcr::TwoColorRadiation, downwelling_shortwave_radiation, coupled_model)
+    ρₒ = coupled_model.interfaces.ocean_properties.reference_density
+    cₒ = coupled_model.interfaces.ocean_properties.heat_capacity
+    J⁰ = tcr.surface_flux
+    Qs = downwelling_shortwave_radiation
+    parent(J⁰) .= - parent(Qs) ./ (ρₒ * cₒ)
```

Comment:
probably we need to multiply this by the sea ice concentration
```suggestion
    parent(J⁰) .= - parent(Qs) ./ (ρₒ * cₒ) .* (1 .- parent(ℵ))
```

Reply:
bummer